Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim as compile-image
+
+WORKDIR /app
+RUN apt-get update && \
+    apt-get -y install gcc mono-mcs && \
+    rm -rf /var/lib/apt/lists/*
+COPY ./requirements.txt /app/requirements.txt
+RUN pip install --no-cache-dir --user -r /app/requirements.txt; rm -r /tmp 
+
```

Comment:
Do you really need a separate compile image? It seems that you are just copying the installed dependencies over

Reply:
I think not,
I can probably use the one already referenced intially